Dockerfile

```
@@ -1,14 +1,17 @@
+# Keep this in sync with the Node version specified for the Conda dev environment (in environment_dev.yml)
 FROM node:18.18-alpine3.18
```

Based on the env descriptors, this will default to `22.x` so let's update here
```suggestion
FROM 22.8.0-alpine3.20
```

---

Could it be:

```suggestion
FROM node:22-alpine3
```

I think I've also seen in the wild:

```suggestion
FROM node:22
```

---

This is the one I was suggesting https://hub.docker.com/layers/library/node/22-alpine3.20/images/sha256-85235be9f710c1ca817cb67892544b7f0cf994ef05380e8ac0d2ea58dce8a988?context=explore 
I messed up when adding the suggestion